game/theater/iadsnetwork/iadsnetwork.py

```diff
@@ -175,10 +196,6 @@ def node_for_tgo(self, tgo: TheaterGroundObject) -> IadsNetworkNode:
                 elif node and group.iads_role == IadsRole.POINT_DEFENSE:
                     # Point Defense Node for this TGO
                     node.add_connection_for_group(group)
-
-        if node is None:
-            # Raise exception as TGO does not participate to the IADS
-            raise IadsNetworkException(f"TGO {tgo.name} not participating to IADS")
```

Comment:
No need for this, we can act accordingly when we check for None. If we would need to raise exceptions, we do it elsewhere, specifically where it wouldn't be considered normal behavior...